editor/src/clj/editor/gui.clj

```diff
@@ -2424,6 +2548,16 @@
   (input texture-names GuiResourceNames :array)
   (output texture-names GuiResourceNames :cached (g/fnk [aux-texture-names texture-names] (into (sorted-set) cat (concat aux-texture-names texture-names))))
 
+  (input aux-material-shaders GuiResourceShaders :array)
+  (input material-shaders GuiResourceShaders :array)
+  (output material-shaders GuiResourceShaders :cached (g/fnk [aux-material-shaders material-shaders material-shader]
+                                                        (into {"" material-shader} (concat aux-material-shaders material-shaders))))
```

Comment:
Hmm, we may need to change this slightly. 🤔

I think with the current code, the default Gui material from the scene referencing a Template might replace the default Gui material inside the referenced scene, or something like that. TBH, I'm not sure what the old behavior was, but we need to check and make sure we still have the same behavior.

After we've established what the correct behavior is, we should be able to alter the outcome simply by changing the order of how values are merged into this map.

Reply:
This is fixed now!